grype/db/v6/vulnerability_store.go

```diff
@@ -0,0 +1,126 @@
+package v6
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"gorm.io/gorm"
+
+	"github.com/anchore/grype/internal/log"
+)
+
+type VulnerabilityStoreWriter interface {
+	AddVulnerabilities(vulns ...*VulnerabilityHandle) error
+}
+
+type VulnerabilityStoreReader interface {
+	GetVulnerability(id int64, config *GetVulnerabilityOptions) (*VulnerabilityHandle, error)
```

Comment:
Should `id` be typedef'd to something, even if it's just `type ID int64`, in case this ever changes or we wanted to pass more context beyond just a db id reference?

Reply:
I'll type these to ID, which I think would make their use case a little more clear, but I don't think we should pass anything other than DB ids for simple getters like this (for instance there is a separate getter by name).